dbfit-java/hsqldb/src/test/java/dbfit/environment/HSQLDBRegressionTest.java

```diff
@@ -0,0 +1,20 @@
+package dbfit.environment;

```

Comment:
I'd edited from Windows Eclipse but checked in from the VM and hadn't set ``autocrlf`` to ``input``.